packages/apisix/apisix.patch

```diff
@@ -0,0 +1,10 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/apisix/charts-original/Chart.yaml packages/apisix/charts/Chart.yaml
+--- packages/apisix/charts-original/Chart.yaml
++++ packages/apisix/charts/Chart.yaml
+@@ -20,3 +20,6 @@
+ name: apisix
+ type: application
+ version: 0.3.3
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: apisix
```

Comment:
Could you please add this additional annotation for the name that will be displayed in the UI?
catalog.cattle.io/display-name: Apache APISIX
I used Apache APISIX as an example, but you can change it to any other name you think is best.

Reply:
Thanks! I agree that Apache APISIX is better.